example.dur

```diff
@@ -35,7 +35,7 @@ a8"    `Y88 88       88 88P'   "Y8 88 ""     `Y8 88P'   `"8a
 
 """
 
-def DurianSwallower(durian : String) : Bool {
+def DurianSwallower(durian) {
     # Durian swallowing never fails baby
```

Comment:
We should probably remove the comments as well since we won't have them for MVP.